readme.md

```diff
@@ -23,10 +23,54 @@ console.log(random(), random(), random());
 
 ## API
 
-### uniqueRandom(minimum, maximum)
+### uniqueRandom(minimum, maximum, options?)
 
 Returns a function, that when called, will return a random number that is never the same as the previous.
 
+#### minimum
+
+Type: `number`
+
+The minimum number that can be returned.
+
+#### maximum
+
+Type: `number`
+
+The maximum number that can be returned.
+
+#### options
+
+Type: `object`
+
+##### exhaustive
+
+Type: `boolean`\
+Default: `false`
+
+Prevent extracted numbers regeneration before all possibilities are exhausted.\
```

Comment:
```suggestion
Prevent extracted numbers regeneration before all possibilities are exhausted.
```

Reply:
Needs better wording

Reply:
Maybe this could be more readable:

```suggestion
Prevent extracted numbers from being re-extracted before all possibilities exhausted, ensuring also the last extracted number can't be the first re-extracted.
```

Reply:
We can do better by leaving out some internal implementation details.

Reply:
Are you intending for example “ensuring also the last extracted number can't be the first re-extracted” for some internal implementation details?

I think instead the definition (first phrase) should be kept so to make sure the user understands well what this option is about, oversimplify may cause the user to misunderstand it.

Reply:
I agree. It needs a better wording.